app/src/main/java/io/gnosis/safe/Tracker.kt

```diff
@@ -57,4 +60,43 @@ class Tracker @Inject constructor(@ApplicationContext context: Context) {
         val PUSH_ENABLED = "enabled"
         val PUSH_DISABLED = "disabled"
     }
+
+    companion object {
+
+        @Volatile
+        private var INSTANCE: Tracker? = null
+
+        fun getInstance(context: Context): Tracker =
+            INSTANCE ?: synchronized(this) {
+                INSTANCE ?: Tracker(context).also { INSTANCE = it }
+            }
+    }
+}
+
+enum class ScreenId(val value: String) {
+    SPLASH("splash"),
+    LAUNCH("launch"),
```

Comment:
@tschubotz changed https://github.com/gnosis/safe/wiki/Overview-screen-tracking (per my request) and we can get rid of splash now and track launch on  the splash screen. If you don't want to do that, I can do it in my PR regarding the SplashScreen.